src/main/scala/viper/gobra/ast/internal/Program.scala

```diff
@@ -806,6 +859,12 @@ case class PermissionT(addressability: Addressability) extends Type {
   override def withAddressability(newAddressability: Addressability): PermissionT = PermissionT(newAddressability)
 }
 
+case object SortT extends Type {
```

Comment:
maybe add a short description of what `SortT` represents. If I'm correct then this is the return type of (ghost) expressions that return the type of an expression, e.g. `TypeOf`

Reply:
But what is the difference to a `TypeExpr`?

Reply:
No, it is just the type of types. Right now we have a single sort, later we might distinguish further.